server/tournaments/index.ts

```diff
@@ -12,7 +12,7 @@ const AUTO_START_MINIMUM_TIMEOUT = 30 * 1000;
 const MAX_REASON_LENGTH = 300;
 const MAX_CUSTOM_NAME_LENGTH = 100;
 const TOURBAN_DURATION = 14 * 24 * 60 * 60 * 1000;
-const ALLOW_ALTS = false;
+const ALLOW_ALTS = !!Config.noipchecks;
```

Comment:
tbh if you could convert all existing `ALLOW_ALTS` to `Config.noipchecks` that'd be great.

Reply:
(Because of the hotpatch thingy)

Reply:
Done.